hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/DAG.java

```diff
@@ -441,25 +441,33 @@ public JsonObject toJson(int defaultLocalParallelism) {
      */
     @Nonnull
     public String toDotString() {
-        return toDotString(LOCAL_PARALLELISM_USE_DEFAULT);
+        return toDotString(LOCAL_PARALLELISM_USE_DEFAULT, DEFAULT_QUEUE_SIZE);
     }
 
     /**
-     * Returns a DOT format (graphviz) representation of the DAG and annotates
-     * the vertices using default parallelism with the supplied value.
+     * Returns a DOT format (graphviz) representation of the DAG, annotates
+     * the vertices using supplied default parallelism value, and the edges
+     * using supplied default queue size value.
+     *
+     * @param defaultLocalParallelism the local parallelism that will be shown if
+     *                                neither overridden on the vertex nor the
+     *                                preferred parallelism is defined by
+     *                                meta-supplier
+     * @param defaultQueueSize the queue size that will be shown if not overriden
```

Comment:
Minor comment: "overriden` contains typo, should be overridden

Reply:
Fixed! Thanks